### ISSUE TITLE
Config not saved after exiting config ui

### ISSUE THREAD

**Issue by S1ss3l** [AnKing-Memberships/AnkiWeekendsAndHolidays#3]
#### Problem description
TL;DR : settings are not saved upon closing config gui and it prevent the addon from working.
Step to reproduce : 

1. Install the add-on from file (latest release, 1.0)
2. Restart anki, go to Anking -> Weekends and Holliday
3. Try to modify anny settings there
4. Hit save
5. Redo step 2, settings are not modified.
Trying to modify the config file directly is of no use. I've searched every add-on file but could not see which one was used as a config file, the config.json file of the root directory is showing default config (wich is what I want) but the file shown by anki when hitting the "advanced" button of the config gui is showing a modified, non working config (see at the end).

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [X] I've restarted Anki to see if it helps
 - [X] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [X] I've tried to disable other add-ons to see if there are any interactions present -> Yea even when anking is the sole addon enabled, bug still persist.
 - [X] My issue disappears when I hold shift while starting Anki. -> Yea, the default config is loaded when I do that.
 - [X] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

Version ⁨2.1.54 (b6a7760c)⁩
Python 3.9.10 Qt 6.3.1 PyQt 6.3.

#### Default config file vs shown config file :
By hitting "reset to default", config file revert back to : 
```
{
    "execute_on_close": false,
    "execute_on_synch": yes,
    "max_change_days": 30,
    "max_days_lookahead": 90,
    "min_interval_days": 0,
    "no_tooltip": false,
    "only_mature_cards": false,
    "reschedule_direction": "both",
    "shortcut": "Ctrl+Shift+w",
    "skip_dates": [],
    "skip_fri": false,
    "skip_mon": false,
    "skip_sat": true,
    "skip_sun": true,
    "skip_thu": false,
    "skip_tue": false,
    "skip_wed": false
}
```

Config file shown in the gui after hitting save and reopening it is : 

```
{
    "execute_on_close": false,
    "execute_on_synch": false,
    "max_change_days": 30,
    "max_days_lookahead": 90,
    "min_interval_days": 0,
    "no_tooltip": false,
    "only_mature_cards": false,
    "reschedule_direction": "both",
    "shortcut": "Ctrl+Shift+w",
    "skip_dates": [],
    "skip_fri": false,
    "skip_mon": false,
    "skip_sat": false,
    "skip_sun": false,
    "skip_thu": false,
    "skip_tue": false,
    "skip_wed": false
}
```
Config.json file :
```
{
    "skip_dates": [],
    "skip_mon": false,
    "skip_tue": false,
    "skip_wed": false,
    "skip_thu": false,
    "skip_fri": false,
    "skip_sat": true,
    "skip_sun": true,
    "only_mature_cards": false,
    "min_interval_days": 0,
    "reschedule_direction": "both",
    "execute_on_synch": true,
    "execute_on_close": false,
    "max_change_days": 30,
    "max_days_lookahead": 90,
    "shortcut": "Ctrl+Shift+w",
    "no_tooltip": false
}
```

Tell me if you need anything else.
Thanks in advance
